index.js

```diff
@@ -1,22 +1,43 @@
+var path = require('path');
+
 module.exports = {
   name: 'ember-cli-6to5',
+
+  treeFor: function(name) {
+    if (name !== 'vendor') { return; }
+
+    // Use the transpiler's node_modules directory the vendor tree for this addon
+    return this.treeGenerator(path.join(__dirname, 'node_modules', 'broccoli-6to5-transpiler', 'node_modules'));
```

Comment:
this will need to use resolve, or the instability of npm's deduping may change this path out from under you.